<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Icons/Icon.png</normaloff>:/Icons/Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="InputTableView" name="TableInput">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="Controller1_Connected">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Controller2_Connected">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Controller3_Connected">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Controller4_Connected">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MapController">
        <property name="text">
         <string>Map Controller 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ComboMapController1"/>
      </item>
      <item>
       <widget class="QLabel" name="MapController_2">
        <property name="text">
         <string>Map Controller 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ComboMapController2"/>
      </item>
      <item>
       <widget class="QLabel" name="MapController_3">
        <property name="text">
         <string>Map Controller 3:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ComboMapController3"/>
      </item>
      <item>
       <widget class="QLabel" name="MapController_4">
        <property name="text">
         <string>Map Controller 4:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ComboMapController4"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckClipCursor">
        <property name="text">
         <string>Clip Cursor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckShowCursor">
        <property name="text">
         <string>Show Mouse Cursor</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InputTableView</class>
   <extends>QTableView</extends>
   <header>InputTableView.moc.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
